StripeCryptoOnramp/StripeCryptoOnramp/Source/Internal/API Bindings/EmptyRequestWithCredentials.swift

```diff
@@ -1,19 +1,19 @@
 //
-//  CustomerRequest.swift
+//  EmptyRequestWithCredentials.swift
```

Comment:
Rather than duplicate the struct for a different request, I’ve just made the name more generic.